src/ServerConfigParams.ts

```diff
@@ -183,15 +177,6 @@ export async function resolveServerConfig(
   );
   /*  } */
 
-  if (
```

Comment:
Why did we remove this check?

Reply:
we have the same check over here, https://github.com/rsksmart/rif-relay-server/blob/0b0935fe5e987afd499f2b3462f5f331b2575593/src/RelayServer.ts#L845. Where should I keep it, or should I keep both?

Reply:
I missed it, totally fine 👍🏽